contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/HdfsFetcher.java

```diff
@@ -167,7 +167,7 @@ private HdfsFetcher(VoldemortConfig config,
      *
      * FIXME: Refactor test code with dependency injection or scope restrictions so this function is not public.
```

Comment:
Thanks a lot for this commit @SidW (:

Now that we're properly passing in the quota as a parameter, rather than getting it from the ```MetadataStore``` instance, I believe we can finally get rid of these two bogus test-only functions.

Do you think you could remove them and refactor the impacted tests accordingly?

-F